test/e2e/cassandracluster_test.go

```diff
@@ -41,7 +47,7 @@ func CassandraClusterTest(code func(t *testing.T, f *framework.Framework,
 	ctx *framework.TestCtx)) func(t *testing.T) {
 	return func(t *testing.T) {
 		ctx, f := mye2eutil.HelperInitOperator(t)
-		defer ctx.Cleanup()
+		//defer ctx.Cleanup()
```

Comment:
any reason why it's commented ?

Reply:
This is just debugging. I am having trouble figuring out how to have resources not get deleted when tests finish running. I will revert that code.